Tests/arm/A001.PSScripts.Test-ARMTemplate.Tests.ps1

```diff
@@ -1,13 +1,13 @@
-# Due to problems with mocking function on the build server this test has been disabled
-
-# Push-Location -Path $PSScriptRoot\..\PSScripts\
-
-Describe "Test-ARMTemplate acceptance tests" -Tag "Acceptance" {
-  
-    It "Should call Test-AzureRmResourceGroupDeployment" {
-        #Mock Test-AzureRmResourceGroupDeployment -ParameterFilter { $ResourceGroupName -and $TemplateFile -and $TemplateParameterFile } -MockWith { return $null }
-        #.\Test-ARMTemplate -TemplateFile template.json -ParameterFile parameters.json
-        #Assert-MockCalled Test-AzureRmResourceGroupDeployment  -Exactly 1
-    }
-
+# Due to problems with mocking function on the build server this test has been disabled
+
+# Push-Location -Path $PSScriptRoot\..\..\PSScripts\
+
+Describe "Test-ARMTemplate acceptance tests" -Tag "Acceptance" {
+  
+    It "Should call Test-AzureRmResourceGroupDeployment" {
+        #Mock Test-AzureRmResourceGroupDeployment -ParameterFilter { $ResourceGroupName -and $TemplateFile -and $TemplateParameterFile } -MockWith { return $null }
```

Comment:
I'm not massively keen on changing any of the tests during this rework but there may be a case for doing it when we've commented out a test rather than using the -Skip switch on It

Reply:
This test is in the same state in master - my change literally just updated the path to the PSScripts folder:

https://github.com/SkillsFundingAgency/dfc-devops/blob/master/Tests/A001.PSScripts.Test-ARMTemplate.Tests.ps1

Reply:
I'll update this to use -Skip instead